### PR TITLE
[T2A2][W15-A3]Sriram Sami

### PR DESCRIPTION
**Pull request by frizensami**

### CODE REVIEW

**Comment by frizensami** [comment]
Ready for review.

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -365,7 +365,8 @@ public static String executeCommand(String userInputString) {
      */
     private static String[] splitCommandWordAndArgs(String rawUserInput) {
         final String[] split =  rawUserInput.trim().split("\\s+", 2);
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:375]: `split` is not a particularly good variable name; you could have changed it to be more reflective of what it is
in any case, its an array, so by the standards, it should have been `splits` :-P

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -365,7 +365,8 @@ public static String executeCommand(String userInputString) {
...
-        return split.length == 2 ? split : new String[] { split[0] , "" }; // else case: no parameters
+        final boolean isThereAnyArguments = split.length == 2;
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:376]: this is a good one!

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -202,13 +202,21 @@ public static void main(String[] args) {
...
-        while (true) {
+        startUserInputLoop();
+    }
+
+    /**
+     * Continually accepts input from the user, processes it, 
+     * and provides visual feedback in an endless loop.
+     */
+	private static void startUserInputLoop() {
+		while (true) {
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:213]: - indentation is off with at `while (true)`

File: src/seedu/addressbook/AddressBook.java [hunk 4]
```diff
@@ -202,13 +202,21 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        startUserInputLoop();
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:205]: - consider a different name as discussed in the tutorial.
- you have a system that is 
  1. welcoming the user
  2. preparing the software for use and load existing data
  3. And, stuck at start?

**Comment by okkhoy** [comment]
@frizensami 
- some good refactoring done here; couple of good catches; 
- i believe you can do even better than this attempt.
- in general, pay more attention to coding standards violation.
- some useful ones include checking how conditional statements are written, naming conventions etc.,

**Comment by frizensami** [comment]
Thanks for the comments! Much appreciated. Apologies - I just saw this.

**Comment by okkhoy** [comment]
welcome! your ACK came in late ;-)

**Comment by okkhoy** [comment]
please close the PR if you haven't done yet.

**Comment by frizensami** [comment]
Yep I think it's closed already

**Pull request closed without merging**